contrib/examples/apiserver/serviceclass.yaml

```diff
@@ -3,3 +3,12 @@ kind: ServiceClass
 metadata:
   name: test-serviceclass
 brokerName: test-broker
+bindable: true
+osbGuid: d35b55b2-b1fd-4123-8045-5b9c619cb629
+plans:
+  - name: example-plan-1
+    osbGuid: 10e03cb7-b2cf-40dd-a954-16a382b92446
+    osbFree: true
+  - name: example-plan-2
+    osbGuid: b8269ab4-7d2d-456d-8c8b-5aab63b321d1
+    osbFree: false
```

Comment:
> "just fyi, this is what one of these things look like, but you don't create it yourself."

That is accurate -- I'll add a README.md to this directory.  Basically these are meant to give you something to play with if you're someone working on integrating with our API server (as I have people doing).